core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -27,14 +27,14 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout title="${it.displayName} Config" cssClass="config j-hide-left" norefresh="true" permission="${it.EXTENDED_READ}">
-    <st:include page="sidepanel.jelly" />
+  <l:layout title="${it.displayName} Config" cssclass="main-panel-only" norefresh="true" permission="${it.EXTENDED_READ}">
     <f:breadcrumb-config-outline />
     <l:main-panel>
-      <l:js src="jsbundles/config-tabbar.js" />
-      <l:css src="jsbundles/jenkins-widgets.css" />
-      <div class="behavior-loading">${%loading}</div>
-      <f:form method="post" action="configSubmit" name="config" tableClass="job-config tabbed">
+      <l:js src="jsbundles/config-scrollspy.js" />
+      <l:css src="jsbundles/config-scrollspy.css" />
+
+      <div class="behavior-loading">${%LOADING}</div>
```

Comment:
I expect this to break all existing translations for no real reason. Could just be changed in the corresponding `.properties` file.